front/app/containers/Admin/sideBar/messages.ts

```diff
@@ -57,6 +57,10 @@ export default defineMessages({
     id: 'app.containers.AdminPage.SideBar.insights',
     defaultMessage: 'Reporting',
   },
+  menu: {
+    id: 'app.containers.AdminPage.SideBar.menu',
+    defaultMessage: 'Pages & menu',
```

Comment:
It always looks like that for all customers (including ones without `customizable navbar`), so it's not always correct.
Not sure yet how to properly solve it.

Reply:
Here (in the core) you should have 'Pages' as the copy. Then to insert 'Pages & menu' (all in the module), have a look at front/app/modules/commercial/insights/index.tsx to see how to insert items in the admin sidebar. Removing the existing one is also something I need more time for. You can search for `navbarModuleActive` in the front codebase. Maybe you'll find your answer there.

Reply:
This is just copy, not actual module functionality, so I think it's fine to go with the simple solution, imo :)